app/services/recurring_contribution/subscriptions/update_status.rb

```diff
@@ -0,0 +1,21 @@
+class RecurringContribution::Subscriptions::UpdateStatus
+  def initialize(subscription_code, current_status)
+    @juntos_subscription = Subscription.find_by(subscription_code: subscription_code)
+    @current_status = current_status
+  end
+
+  def self.process(subscription_id, current_status)
+    new(subscription_id, current_status).process
+  end
+
+  def process
+    update_juntos_subscription_status
+  end
+
+  private
+
+  def update_juntos_subscription_status
```

Comment:
Achei esse método meio inútil. Você tá delegando algo de 2 linhas e tal. Não sei se mais alguém concorda comigo...